source/query-bar.txt

```diff
@@ -329,3 +312,43 @@ as favorites. To view the list of queries saved as favorites, click on
       :titlesonly:
 
       /export-query-to-language
+
+Limitations
+-----------
+
+Limitations apply to the following query operators:
```

Comment:
So `$slice` isn't technically a query operator, it's a projection operator (https://docs.mongodb.com/manual/reference/operator/projection/). I think we can drop `query` and just say `the following operators`.

If we end up adding more limitations to this section down the line, it may make sense to split this up into subsections based on operator type, but for now I think it's fine to leave general.

Reply:
Agreed. I considered splitting the list by type somehow, but it seemed too small a list to warrant it.